src/store/modules/bitcoin.js

```diff
@@ -118,7 +118,9 @@ const mutations = {
     const mergedBlocks = [...blocks, ...state.blocks];
     // remove duplicate blocks
     const uniqueBlocks = mergedBlocks.filter((v, i, a) => a.findIndex(t => (t.height === v.height)) === i);
-    state.blocks = uniqueBlocks;
+
+    // limit to latest 6 blocks
+    state.blocks = [...uniqueBlocks.slice(0, 6)];
```

Comment:
I don't think this is the right solution, because the Bitcoin view needs the whole blocks array to calculate sync percentage.

Reply:
I'll limit it to just where the view is using it. Good call

Reply:
But it has been asked several time to calculate sync percent based on downloaded blockchain size and not current block height (#242, #225).

Reply:
> because the Bitcoin view needs the whole blocks array to calculate sync percentage here.

Oh no, we do not need the entire blockchain on the front-end. The sync % is separately fetched from the back-end, so it's safe to discard the blocks on the front-end.

Reply:
Oops, sorry 😁